src/utils.rs

```diff
@@ -69,6 +69,10 @@ pub enum HassleError {
     CompileError(String),
     #[error("Validation error: {0}")]
     ValidationError(String),
+    #[error("Error loading library")]
+    LoadLibraryError(#[from] libloading::Error),
```

Comment:
Is the underlying error printed in this case, or is that only applicable to `#[error(transparent)]`?

Reply:
Updated, because this also wouldn't print the filename of the DLL / SO that was being attempted.